### DISCUSSION TITLE
how to let mastodon run on different port then 443

### DISCUSSION THREAD

**Discussion by frontibit** [mastodon/mastodon#20279] [General]
I try to let mastodon run on a port 5443 instead of 443 
Do i need to change in .enc.production the LOCAL_DOMAIN as well?
i have modified the ngnix config like this:
server {
  listen 5443 ssl http2;
  listen [::]:5443 ssl http2;
...
and the proxy settings:

  location @proxy {
    proxy_set_header Host $host:5443;
    proxy_set_header X-Real-IP $remote_addr;

is theer anything else to do as its not working in terms of communication to other hosts.
I can search and find other users on other servers but not able to follow them..

is there somewhere a good documentation telling about Port and communication settings?

**Comment by MrGeneration** [comment]
I believe that's not possible as other feediverse servers can't know that they should use different ports than the default https port.

**Reply by frontibit** [reply]
ok understand but sad :)
